python/ynab_client.py

```diff
@@ -65,6 +67,3 @@ def findPreviousTransaction(payee_name):
         .filter(Transaction.imported_payee==payee_name)\
         .order_by(Transaction.date.desc())\
         .first()
-
-# Sync with YNAB
-sync()
```

Comment:
The client should be imported at that point, all files checked syntactically and compiled, and the app about to run.